nordigen/api/accounts.py

```diff
@@ -85,8 +85,36 @@ def get_transactions(self, date_from: Optional[str] = None, date_to: Optional[st
         Returns:
             Dict: account transactions details
         """
-        date_range = {
-            "date_from": date_from,
-            "date_to": date_to
-        }
-        return self.__get("transactions", date_range)
+        date_range = {"date_from": date_from, "date_to": date_to}
+        return self._get("transactions", date_range)
+
+
+class PremiumAccountApi(AccountApi):
```

Comment:
I think it is better not to inherit `AccountApi` but create a new file `nordigen/api/accounts_premium.py`, as the class `AccountApi` is not very open for this modification. Overwriting `_get` violates the Open/Closed Principle.